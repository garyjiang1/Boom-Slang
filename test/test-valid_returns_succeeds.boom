def my_func(int x) returns int:
	if x > 5:
		println("hey")
	else:
		return 5
	return 1

println(my_func(7))

println(my_func(5))
