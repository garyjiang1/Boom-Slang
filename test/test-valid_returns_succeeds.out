hey
1
5
